paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerAdapter.kt

```diff
@@ -44,20 +44,20 @@ interface CustomerAdapter {
 
     /**
      * Saves the payment option to a data store.
-     * @param paymentOption, the [PersistablePaymentMethodOption] to save to the . If
-     * null, the selected payment method option will be cleared from the data store.
+     * @param paymentOption, the [PaymentOption] to save to the data store. If null, the selected
+     * payment method option will be cleared from the data store.
      */
-    suspend fun setSelectedPaymentMethodOption(paymentOption: PersistablePaymentMethodOption?)
+    suspend fun setSelectedPaymentOption(paymentOption: PaymentOption?)
 
     /**
-     * Retrieves the saved payment option from a data store. If null, the customer does not have a
-     * default saved payment method.
+     * Fetch the saved payment method option from a data store. If null, the customer does not have
+     * a default saved payment method.
      */
-    suspend fun fetchSelectedPaymentMethodOption(): Result<PersistablePaymentMethodOption?>
+    suspend fun fetchSelectedPaymentOption(): Result<PaymentOption?>
```

Comment:
I still think we should rename this to `retrieveSelectedPaymentOption`. But happy to punt this to a follow up.